README.md

```diff
@@ -8,13 +8,7 @@ explore and build. The best way to share this is via our [MongoDB Community Foru
 
 The development version of this package supports Django 5.0.x. To install it:
 
-`pip install django-mongodb-backend`
-
-### Resources
-
- Check back soon as we aim to provide more links that will dive deeper into our library!
-
-* [Developer Notes](DEV_NOTES.md)
+`pip install django-mongodb-backend~=5.0.0`
```

Comment:
I don't believe this works (or is what we want people to use). Did you see the version I proposed?
```
$ pip install django-mongodb-backend~=5.0.0
ERROR: Could not find a version that satisfies the requirement django-mongodb-backend~=5.0.0 (from versions: 5.0.0a1, 5.0.0a2, 5.0.0a3)

[notice] A new release of pip is available: 24.3.1 -> 25.0
[notice] To update, run: pip install --upgrade pip
ERROR: No matching distribution found for django-mongodb-backend~=5.0.0
```

Reply:
s/~=/==/

Reply:
Hadn't seen all the comments yet. Reviewing everything now and applying appropriate changes.